e2e/testcafe-devextreme/tests/accessibility/menu.ts

```diff
@@ -42,17 +42,24 @@ const options: Options<Properties> = {
   items: [items],
   disabled: [true, false],
   orientation: ['horizontal', 'vertical'],
+  adaptivityEnabled: [true, false],
 };
 
 const created = async (t: TestController, optionConfiguration): Promise<void> => {
-  const { disabled } = optionConfiguration;
+  const { disabled, adaptivityEnabled } = optionConfiguration;
 
   if (disabled) {
     return;
   }
 
   const menu = new Menu();
 
+  if (adaptivityEnabled) {
+    await menu.option('width', 400);
```

Comment:
Is far as i see we don't need changing it in runtime, we can just add `width: [400, 1024]` in options object

Reply:
[5928c7e](https://github.com/DevExpress/DevExtreme/pull/28066/commits/5928c7ef8aab472b03dce8be2f2d2f02cefed50d)